programs/cltester/dwarf.cpp

```diff
@@ -928,8 +1176,221 @@ namespace dwarf
          parse_debug_info_unit(result, whole_s, unit_s, {s.pos, s.pos + unit_length});
          s.skip(unit_length);
       }
+      std::sort(result.subprograms.begin(), result.subprograms.end());
+      for (size_t pos = 0; pos < result.subprograms.size();)
+         pos = fill_parents(result, pos, pos + 1);
+   }
+
+   Elf64_Word add_str(std::vector<char>& strings, const char* s)
+   {
+      if (!s || !*s)
+         return size_t(0);
+      auto result = strings.size();
+      strings.insert(strings.end(), s, s + strlen(s) + 1);
+      return result;
    }
 
+   template <typename T>
+   struct fill_later
+   {
+      size_t* offset = nullptr;
+   };
```

Comment:
I don't see anything that creates an attribute with this kind of value.

Reply:
The code that used it is no longer present. removed.